resources/test/bedcounts.csv

```diff
@@ -0,0 +1,2 @@
+icu_name,n_covid_occ,n_covid_free,n_ncovid_occ,n_ncovid_free,n_covid_deaths,n_covid_healed,n_covid_refused,n_covid_transfered,create_date
+hopital3,11,12,13,14,15,16,17,18,2020-04-16T23
```

Comment:
What is 16T23?

Reply:
It's 11pm on the 16th.

Reply:
Ah ok, is this a common format? Is that what we want in our tests?

Reply:
No strong opinion, I changed it to something more common.